.github/workflows/rust.yml

```diff
@@ -239,7 +239,9 @@ jobs:
           name: risc-v-esp32c6-static-library
           path: c-library/risc-v-esp32-c6/target/riscv32imac-unknown-none-elf/release
       - name: Build
-        run: cd ./examples/c-examples/risc-v-esp32c6 && . /root/esp/esp-idf/export.sh && make && esptool.py --chip esp32c6 elf2image --flash_mode="dio" --flash_freq "40m" --flash_size "4MB" -o main.bin main.elf
+        run: |
+          sed -i 's/0x4086E610/0x4086E210/' "$IDF_PATH/components/esp_system/ld/esp32c6/memory.ld.in" &&
```

Comment:
Something redundant

Reply:
In esp32c6.ld in point 3 it is written like this:
![image](https://github.com/user-attachments/assets/790fcc17-a7a0-4203-a020-15e898bb4546)

Still not necessary?